pages/osx/mate.md

```diff
@@ -0,0 +1,24 @@
+# mate
+
+> General-purpose text editor for macOS.
+> More information: <https://macromates.com/>.
+
+- Start TextMate:
+
+`mate`
+
+- Open specific files:
+
+`mate {{path/to/file1 path/to/file2 ...}}`
+
+- Specify the filetype of a file:
+
+`mate --type {{filetype}} {{path/to/file}}`
+
+- Open and wait until done editing a specific file:
```

Comment:
```suggestion
- Open and wait until finishing editing a specific file:
```

Reply:
This exact line was taken directly from [the `webstorm` tldr-page](https://github.com/tldr-pages/tldr/blob/da55f621eb9c77d579b4353d78cc4c9bec39c4f3/pages/common/webstorm.md), but I agree that "finishing" is better than "done".

I think "finished" might be a better form of the word than "finishing" to use here, though.